doc/examples/rendering/perLocationShaderVariationArnold.gfr

```diff
@@ -442,35 +441,52 @@ Gaffer.Metadata.registerValue( __children["Scene"]["BoxOut"]["__out"], 'nodule:t
+__children["Backdrop4"]["title"].setValue( 'Per-location color variation' )
+__children["Backdrop4"]["description"].setValue( "Per-object variation can be achieved by using the CustomAttributes node to randomize a color based on each object's location, and then reading that resulting color value into a shader at render time.\n\nFirst, we add a custom `user:colorVariation` Color3f attribute to the CustomAtributes node. We then drive that attribute with a Random node (right-click the plug, then select Randomise... from the context menu).\n\nThis creates a new attribute (which you can verify in the Scene Inspector), but with a single color shared across all locations. In order to make it vary, we need to tell the node to change its value based on the object's location (also known as path).\n\nTo do this, we make use of the Random node's Context Entry plug. This tells the node to generate a new random value whenever some part of the context changes. If you're not familiar with the concept of contexts in Gaffer, in a nutshell they contain information about which location and frame a node is currently computing.\n\nHere, we set the Context Entry plug to `scene:path`, which tells the node to generate a new random color for each location in the scene. Since each cow in the scene has a different location (or path), each should thus have a unique color.\n\nYou can verify this in the Scene Inspector (by default in the tab next to the Node Editor). If you select a cow in the Viewer or Hierarchy View, and then expand the Attributes section in the Scene Inspector, you should see its `user:colorVariation` attribute. Each cow you select should have a different value for that attribute." )
 __children["Backdrop4"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -10, -37.5400391 ), imath.V2f( 48.2769547, 9.48580933 ) ) )
 __children["Backdrop4"]["__uiPosition"].setValue( imath.V2f( -112.693184, 37.8769875 ) )
-Gaffer.Metadata.registerValue( __children["Backdrop5"], 'nodeGadget:color', imath.Color3f( 0.400000006, 0.360000014, 0.360000014 ) )
-__children["Backdrop5"]["title"].setValue( 'Debugging' )
-__children["Backdrop5"]["description"].setValue( 'You can use the AttributeVisualiser node to see the per-location values directly in the Viewport!' )
-__children["Backdrop5"]["__uiBound"].setValue( imath.Box2f( imath.V2f( 7.30519485, -5.54041862 ), imath.V2f( 31.8014202, 10.5616188 ) ) )
-__children["Backdrop5"]["__uiPosition"].setValue( imath.V2f( -25.7214622, -17.3105278 ) )
+Gaffer.Metadata.registerValue( __children["Backdrop5"], 'nodeGadget:color', imath.Color3f( 0.0691713095, 0.5, 0.414892226 ) )
+__children["Backdrop5"]["title"].setValue( 'Tip: Previewing the color variation' )
```

Comment:
Could we tone down this backdrop color a little? It seems quite saturated compared to all the others?

Reply:
2f861a2cba670582a063d0cb98a09bf9f6764cd5